### DISCUSSION TITLE
Hide forks of public projects when profile is set to private

### DISCUSSION THREAD

**Discussion by jhogberg** [community/community#102136] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When setting one's profile to private, there is currently no way to hide a fork of a public repository. While I'm aware that you can delete a fork once you are done contributing (e.g. in between PRs), that doesn't help at all for regular contributors to a project: as long as you keep sending PRs, the repository will stay on your profile page.

Please consider adding a setting to hide forks of public repositories from the profile page (either all of them or a selection). 

Another variant, for which I found a [related discussion](https://github.com/orgs/community/discussions/59987), is to allow making _private forks_ of public repositories that can only be used for submitting pull requests (that is, it cannot be browsed).

**Comment by gopal-jogi** [comment]
GitHub doesn't provide a direct way to selectively hide forks of public repositories on your profile. You may consider creating a separate account for personal contributions. The idea of private forks for pull requests only is interesting but currently not implemented.
